querybook/webapp/const/metastore.ts

```diff
@@ -74,7 +74,7 @@ export interface IDataTable {
         partition_keys?: string[];
     };
     custom_properties?: Record<string, string | number>;
-
+    table_links?: Record<string, string>[];
```

Comment:
it looks like a link can only have `label` and `url`, can we define a new type for it with `label` and `url`, while `label` as optional?

Reply:
+1, using Record is not clear